src/renderer/src/pass/deferred.rs

```diff
@@ -150,7 +150,8 @@ pub static LIGHT_FRAGMENT_SRC: &'static [u8] = b"
             float dist = length(delta);
             float inv_dist = 1. / dist;
             vec4 light_to_point_normal = delta * inv_dist;
-            float intensity = dot(light[i].propagation.xyz, vec3(1., inv_dist, inv_dist * inv_dist));
+            // float intensity = dot(light[i].propagation.xyz, vec3(1., inv_dist, inv_dist * inv_dist));
```

Comment:
Sorry, that was just me playing around with the deferred rendering. I've updated this line in the latest commit